pwncat/platform/linux.py

```diff
@@ -621,6 +621,9 @@ def __init__(self, session, channel: pwncat.channel.Channel, *args, **kwargs):
             self.shell = "/bin/sh"
             self.channel.sendline(b" export SHELL=/bin/sh")
 
+        if True or self._do_which("which") is None:
```

Comment:
Hey @calebstewart,

you sure that `if True` should be there?

Reply:
You are totally right. I was doing some last minute testing before I pushed this last night and left that in by mistake. Fixing now. Thanks!